runtime/sam/expr/agg/avg.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	"github.com/brimdata/super"
-	"github.com/brimdata/super/runtime/sam/expr/coerce"
+	"github.com/brimdata/super/runtime/sam/expr/cast"
```

Comment:
Nit: I think you should leave this stuff in runtime/sam/expr/coerce. "Coerce" is a more evocative name than "cast". And it's one less package.